authclient/OrcidClient.php

```diff
@@ -57,6 +57,7 @@ public function init()
             $this->scope = $config->scopes;
             $this->clientId = $config->clientId;
             $this->clientSecret = $config->clientSecret;
+            $this->orcidAttribute = $config->orcidAttribute;

```

Comment:
Where there is no `public $orcidAttribute` this is not possible to be used it should either be removed or defined properly before use.